src/main/python/cfn_sphere/__init__.py

```diff
@@ -33,9 +33,6 @@ def create_or_update_stacks(self):
             raw_template = FileLoader.get_file_from_url(stack_config.template_url, stack_config.working_dir)
             template = CloudFormationTemplateTransformer.transform_template(raw_template)
 
-            combined_tags = dict(self.config.tags)
-            combined_tags.update(stack_config.tags)
-
```

Comment:
Yes indeed. I think it is much easier to follow the code if the tags are merged as early as possible. This also underlines the fact that they don't change during runtime.